system/metal-settings/bios_versions/lenovo/thinksystem-sr850p/4_30_tee186p.yaml

```diff
@@ -0,0 +1,4 @@
+version: "4.30 (Build ID: TEE186P)"
+image:
+  URI: "https://repo.eu-de-1.cloud.sap/hw-firmware/lenovo/thinksystem-sr850p/lnvgy_fw_uefi_tee186p-4.30_anyos_32-64.uxz"
```

Comment:
this exposes an internal system in a public helm chart and should be avoided via values